connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClient.scala

```diff
@@ -355,6 +389,11 @@ object SparkConnectClient {
 
     def sslEnabled: Boolean = _configuration.isSslEnabled.contains(true)
 
+    def retryParameters(parameters: RetryParameters): Builder = {
```

Comment:
Can we match the name with the Python side? Seems like it's being called `retryPolicy`.  This kind of thing is important when you fix a same problem across multiple client. Otherwise, you have to read the different code that works for the same to fix the same.

Reply:
Agreed!